packages/bvaughn-architecture-demo/components/console/MessageHoverButton.tsx

```diff
@@ -72,7 +72,10 @@ export default function MessageHoverButton({
           ref={ref}
         >
           <Icon className={styles.AddCommentButtonIcon} type="comment" />
-          <span className={isHovered ? styles.LabelHovered : styles.Label}> Add comment </span>
+          <span className={isHovered ? styles.LabelHovered : styles.Label}>
+            {" "}
+            Add comment &nbsp;
```

Comment:
This feels a little hacky. Why two spaces on the right side?

![Screen Shot 2022-08-10 at 10 50 35 AM](https://user-images.githubusercontent.com/29597/183934323-cdd7e57a-1432-49e4-96e8-3d9f1cba3d09.png)

If you're aiming to horizontally balance the padding on the right side with the icon on the left, can we just use `padding-right`?

Reply:
Yeah, this was weird, I was having trouble getting padding to work yesterday, but got things fixed up.